.github/workflows/ci.yml

```
@@ -1,27 +1,37 @@
 name: CI
-'on':
+
+on:
   push:
     paths-ignore:
-      - docs/**
+      - 'docs/**'
       - '*.md'
   pull_request:
     paths-ignore:
-      - docs/**
+      - 'docs/**'
       - '*.md'
+
 jobs:
+  dependency-review:
+    name: Dependency Review
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+
+      - name: Dependency review
+        uses: actions/dependency-review-action@v1
+
   test:
     runs-on: ${{ matrix.os }}
+    permissions:
+      contents: read
     strategy:
       matrix:
-        node-version:
-          - 10
-          - 12
-          - 14
-          - 16
-        os:
-          - macos-latest
-          - ubuntu-latest
-          - windows-latest
+        node-version: [10, 12, 14, 16, 18]
```

```suggestion
        node-version: [14, 16, 18]
```

I think at this point 10 and 12 can be removed. Neither Fastify supports 10 and 12.